scripts/publish.js

```diff
@@ -17,16 +17,31 @@ shell.set('+v');
  *
  */
 
-// Checks that you are running this with Node v10.2.0 and above
+// Checks that you are running this with Node v12.4.0 and above
 const [version, major, minor, patch] = process.version.match(
   /^v(\d+)\.?(\d+)\.?(\*|\d+)$/
 );
-if (parseInt(major) !== 10 || parseInt(minor) < 2) {
+if (parseInt(major) !== 12 || parseInt(minor) < 4) {
   console.log(
-    'You do not have the right version of node. We require version 10.2.0.'
+    'You do not have the right version of node. We require version 12.4.0.'
   );
   process.exit(-1);
 }
+// Check if you have installed all the required tooling
```

Comment:
All these dependencies are installed globally but we didn't use to check for them. This will give the user a better error message when the script fails and is caused by one of these dependencies not being installed.